spec/controllers/bulk_updates_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe BulkUpdatesController, reset_provider: true do
+  # TODO add contexts 'when bulk updates enabled' and 'when bulk updates disabled' ?
```

Comment:
Yes, please. Make sure that users cannot access this controller if bulk edits are not enabled. Can easily be copied from bulk updates search controller spec.